### PR TITLE
[WIP] Add option to show timestamps in terminal

### PR DESCRIPTION
**Pull request by MrBurrBurr**
- [ ] Keep timestamp in message and render it as time string
- [ ] Make all messages in format `{ type, timestamp, message }` and decide whether to render the time string or not in template

### CODE REVIEW

File: src/views/Commands.vue [hunk 1]
```diff
@@ -73,7 +73,9 @@
 				log: [],
 				commandHistory: new CommandsCache(20),
 				commandHistoryIndex: -1,
-				asfCommands: []
+				asfCommands: [],
+				timestamp: '',
```
Comment (Aareksio) [inline src/views/Commands.vue]: Why would we need a timestamp as part of the component state?

File: src/views/Commands.vue [hunk 2]
```diff
@@ -73,7 +73,9 @@
...
+				showTimestamps: storage.get('settings:timestamps')
```
Comment (Aareksio) [inline src/views/Commands.vue]: I don't think this is component state either

File: src/views/Commands.vue [hunk 3]
```diff
@@ -280,6 +284,13 @@
 				}
 
 				return this.fetchCommands();
+			},
+			getTimestamp() {
```
Comment (Aareksio) [inline src/views/Commands.vue]: We should not prepare timestamp strings ahead of time, instead compose them on render

Comment (MrBurrBurr) [reply src/views/Commands.vue]: I don't understand what you mean sry.

Right now I get a timestamp each time i write something to the terminal. Is that not how it should be?

**Pull request closed without merging**